nextflow_schema.json

```diff
@@ -37,6 +37,26 @@
                     "description": "Release name (specify TEST.consortium/TEST.public to test consortium and public releases). E.g: 13.1-consortium",
                     "default": "TEST.consortium",
                     "pattern": "\\d+[.]\\d+-(consortium|public)$|TEST.public|TEST.consortium"
+                },
+                "main_pipeline_docker":{
+                    "type": "string",
+                    "description": "Docker used in main processes - validate_data, process_main, process_maf, create_consortium_release, and create_public_release",
+                    "default": "sagebionetworks/genie:latest"
```

Comment:
Should we use the ghcr image?

Reply:
The GHCR image hasn't been built off `main` yet until we do a release. I think we can create another PR to update this after that's done.